src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -958,8 +958,10 @@ def set_deployment_user(user_name, password=None):
             raise CLIError('Please specify both username and password in non-interactive mode.')
 
     user.publishing_password = password
-    result = client.update_publishing_user(user)
-    return result
+    try:
```

Comment:
you can do it in the commands.py, e.g., cli_command(__name__, 'webapp deployment user set', custom_path + 'set_deployment_user', `exception_handler=ex_handler_factory()`)